anta/cli/nrfu/__init__.py

```diff
@@ -57,6 +57,15 @@ def parse_args(self, ctx: click.Context, args: list[str]) -> list[str]:
 @click.pass_context
 @inventory_options
 @catalog_options
+@click.option(
+    "--expand-atomic",
+    "-x",
+    default=False,
+    show_envvar=True,
+    is_flag=True,
+    show_default=True,
+    help="Flag to indicate if atomic results should be rendered",
```

Comment:
Will we support all `anta nrfu` commands (csv, json, etc.)? We should be more specific here.

Reply:
I moved this under `anta nrfu` for now.
We support Table, JSON and Jinja for now. Only Table required this option.